src/rtp/ext.rs

```diff
@@ -356,6 +342,70 @@ impl ExtensionMap {
 
         orig_len - b.len()
     }
+
+    pub(crate) fn narrow(&mut self, remote_exts: &[(u8, Extension)], is_audio: bool) {
+        // Match remote numbers, this does not remove any configured extension.
```

Comment:
This seems too accepting, we should verify that the remote hasn't changed its mind or violated the invariants(same ext id mapping to the same header extension).

Reply:
Ah yes. This is where I mean we should add more SDP validation earlier. To detect such shenanigans.